boilerplate/app/models/Episode.test.ts

```diff
@@ -27,14 +27,14 @@ const episode = EpisodeModel.create(data)
 test("publish date format", () => {
   expect(episode.datePublished.textLabel).toBe("Jan 20, 2022")
   expect(episode.datePublished.accessibilityLabel).toBe(
-    'demoPodcastListScreen:accessibility.publishLabel {"date":"Jan 20, 2022"}',
+    "demoPodcastListScreen:accessibility.publishLabel",
```

Comment:
Can you explain please why are we removing the date? Read from your PR description, we want to still improve this test later.

Reply:
I don't really understand why this changed, but I also didn't want to block this on this test, because the behavior is correct when I actually test it in the app.

Reply:
The single quotes changing to double quotes was required by the linter, and I think might be related to tests having a separate TS config? but not sure, I thought that would be handled by prettier.

Actually I'm even more confused by this because apparently it failed on the CLI project's prettier check (https://app.circleci.com/pipelines/github/infinitered/ignite/3631/workflows/6e7e16e3-e4c1-4a6b-949b-d1fe26da1665/jobs/4375), but the inner project isn't complaining about the double-quotes.

That config just isn't unified, so we get weird things like this sometimes.

Reply:
My question was not related to quotes but related to deleting the date within it.

Reply:
Yes, for that part see my first comment. I'm not sure why it changed, but I'd also like to not block this PR on that since the logic is working correctly on the real app, this is just a test that I'm not sure is testing anything useful the way it's currently set up.

Reply:
Filed issue for this: https://github.com/infinitered/ignite/issues/2789